### PR TITLE
add OBI D error monitoring and injection

### PR DESCRIPTION
**Pull request by strichmo**
Also updates the RTL hash on to E40X

Enables replications of issues with OBI D NMI handler entry

### CODE REVIEW

File: lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv
Comment (MikeOpenHWGroup) [inline lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv:180]: Unless this is still needed, recommend it be removed before merging.

File: lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv
Comment (MikeOpenHWGroup) [inline lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv:208]: Same here: recommend removing all calls to $display() before merging, or re-coding as uvm_info().

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
Interesting.  It appears that you are either taking advantage of, or compensating for, know behaviour of the RM - is that the case?

I asked for a couple of minor cleanups before merging.

**Comment by strichmo** [comment]
Thanks for review @MikeOpenHWGroup 
Moving this to a draft and setting DO NOT MERGE.  

Lee is pushing in a need update to OVPSIM and then I will address the above and clean up the PR to get to a clean data bus directed test.

File: lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv
Comment (strichmo) [reply lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv:180]: cleaned up

File: lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv
Comment (strichmo) [reply lib/uvm_agents/uvma_rvvi_ovpsim/seq/uvma_rvvi_ovpsim_obi_seq.sv:208]: Cleaned up

**Comment by strichmo** [comment]
Dropping this branch.  The work will be merged into a future branch released in combination with the OVPSIM modeling update.

**Pull request closed without merging**
